nemo/core/classes/modelPT.py

```diff
@@ -791,6 +793,14 @@ def setup_optimization(self, optim_config: Optional[Union[DictConfig, Dict]] = N
         if lr is not None:
             optimizer_args['lr'] = lr
 
+        if 'var_noise' in optimizer_args:
+            self._var_noise_std = optimizer_args['var_noise'].get('std', None)
+            self._var_noise_start = optimizer_args['var_noise'].get('start_step', 0)
```

Comment:
Rename these to `_optim_variational_noise_std` and `_optim_variational_noise_start` since its part of optim inside ModelPT